src/main/java/com/openlattice/client/serialization/SerializationConstants.java

```diff
@@ -193,8 +197,10 @@
     public static final String ORGANIZATION_ID                  = "organizationId";
     public static final String ORIENTATION                      = "orientation";
     public static final String PAGING_TOKEN                     = "pagingToken";
+    public static final String PATH                             = "path";
```

Comment:
this should be after `PARTITIONS` in the interest of alphabetization

Reply:
whoops